base/markdown/Julia/interp.jl

```diff
@@ -39,7 +39,7 @@ end
 
 toexpr(x) = x
 
-toexpr(xs::Vector{Any}) = Expr(:cell1d, map(toexpr, xs)...)
+toexpr(xs::Vector{Any}) = Expr(:vectany, map(toexpr, xs)...)
```

Comment:
Yeah, but let's leave that for another PR, since I don't think I would be able to do that.